os/pm/pm.h

```diff
@@ -132,6 +132,9 @@ struct pm_global_s {
 	/* Timer to decrease state */
 
 	WDOG_ID wdog;
+
+	/* No. of Registered Domains */
+	uint32_t ndomains;
```

Comment:
uint32 is too large for ndomains , i think it can be uint8 or uint16

Reply:
Done